### ISSUE TITLE
Cleanup menuitemradio grouping author requirements

### ISSUE THREAD

**Issue by scottaohara** [w3c/aria#1275]
Per a question on web a11y slack yesterday, the `menuitemradio` description first says authors MAY use a `separator` to divide the menu items (including radios) into groups.  The description then goes on to say that authors SHOULD use a `group` and `separator`s to delimit `menuitemradio` groups from other `menuitem`s.  

It's confusing that the spec is first saying that an author MAY use a separator, only to then be told  they SHOULD use a group and also SHOULD use the separator as well when delimiting menu item radio groups.

Part of this contradiction appears to come from the similar description all the menu item roles have... but `menuitemradio` is also pulling from / trying to stay in sync with content from `radiogroup`.

I purpose (and will submit a PR for this) that authors MAY use separators to delimit menuitemradio groups, but SHOULD use `group`.  Also, rather than having two sentences that state this MAY allowance for `separator`, I aim to combine them into a single sentence.
